apps/hash-api/src/index.ts

```diff
@@ -586,7 +586,7 @@ const main = async () => {
       }).pipe(
         Effect.provide(
           RpcClient.connectLayer(
-            Transport.multiaddr(`/dns/${rpcHost}/tcp/${rpcPort}`),
+            Transport.multiaddr(`/dns4/${rpcHost}/tcp/${rpcPort}`),
```

Comment:
What would be required to allow `dns`? shouldn't it be enough to enable listening on `IPv4` on the graph? Is it worth trying out to use `/dns/` and see if it now properly connects to the IPv4 under the assumption that everything else is working now?

Reply:
I'd say we enable IPv6 in the Graph then rather than restricting it to IPv4